cs130_front/src/components/views/GroupsPage/GroupItem.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import Text from '../../Text/Text';
+import TextInput from '../../TextInput/TextInput';
+import Button from '../../Button/Button';
+import UserList from '../../UserList/UserList';
+import * as Colors from '../../../constants/Colors';
+import './styles.css';
+
+const members = [
+    { name: "Shirly fang", school: "UCLA", id: 123, discord: "shirly#123", email: "shirly@gmail.com" },
+    { name: "Shirly fang", id: 123, discord: "shirly#123", email: "shirly@gmail.com" }
+]
+
+export default function GroupItem(props) {
```

Comment:
function GroupItem(props)

Reply:
When I do this, it leads to errors since I can't import GroupItems in the GroupsPage